source/_integrations/enphase_envoy.markdown

```diff
@@ -36,10 +36,23 @@ This integration will offer various sensors depending on the configuration of yo
 - Historical energy production & consumption
 - Power production per-inverter
 
-Production and consumption sensors for each phase are available for Envoy S Metered / IQ Gateway Metered with installed and configured current transformers (CT) on more than 1 phase.
-
 _Consumption sensors require your Envoy to be properly configured with consumption CT sensors installed._
 
+For Envoy S Metered / IQ Gateway Metered with installed and configured current transformers (CT), additional features are available:
+
+- Production and consumption sensors for each phase, if CT are installed on more than 1 phase.
+- Sensors for net production (grid export) and net consumption (grid import) if the consumption CT is a net-consumption CT.
+- Disabled sensors for:
+  
+  - Phase net production and net consumption.
+  - Frequency Net consumption CT (Aggregate and phase).
+  - Voltage Net consumption CT (Aggregate and phase).[^1]
+  - Metering Status for net consumption and production CT (`normal` | `not-metering` | `check-wiring`) (Aggregate and phase).
+  - Meter status flags active for net consumption and production CT (Aggregate and phase).[^2]
+
+[^1]: For multi-phase system the Envoy sums the voltages of the phases. May be valid for split-phase, but for 3 phase systems use the individual phases rather then the summed value.
```

Comment:
```suggestion
[^1]: For multi-phase systems, the Envoy sums the voltages of the phases. May be valid for split-phase, but for 3-phase systems use the individual phases rather than the summed value.
```